pkg/pipeline/encode/encode_json.go

```diff
@@ -1,51 +0,0 @@
-/*
```

Comment:
Now that all the encode and write stages also return GenericMap, the encode_json is no longer needed, since this must now be done anyway by any encode or write stage.